tests/basic_server.py

```
@@ -0,0 +1,110 @@
+# Lists For Clients and Their Nicknames
+clients = []
+nicknames = []
+
+# Sending Messages To All Connected Clients
+def broadcast(message, _client=None):
+    for client in clients:
+        # Send to all clients except the one specified (optional)
+        if client != _client:
+            try:
+                client.send(message)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)